tests/baselines/reference/quickInfoDisplayPartsClassDefaultAnonymous.baseline

```diff
@@ -0,0 +1,76 @@
+[
+  {
+    "marker": {
+      "fileName": "/tests/cases/fourslash/quickInfoDisplayPartsClassDefaultAnonymous.ts",
+      "position": 0,
+      "name": "1"
+    }
+  },
+  {
+    "marker": {
+      "fileName": "/tests/cases/fourslash/quickInfoDisplayPartsClassDefaultAnonymous.ts",
+      "position": 7,
+      "name": "2"
+    },
+    "quickInfo": {
+      "kind": "class",
+      "kindModifiers": "export",
+      "textSpan": {
+        "start": 7,
+        "length": 7
+      },
+      "displayParts": [
+        {
+          "text": "class",
```

Comment:
`class default` is weird but correct, at least according to commonjs:

1. assignment to a variable gives an anonymous function the name of the variable, eg `var f = function () { }`
2. JS classes are just functions.
3. `export default x` translates to `exports.default = x` in commonjs.

I think this is in the standard, but at least node and TS agree on this.

Reply:
Node appears to agree even in ESM mode:

**foo.mjs**
```js
export default class {}
```

**bar.mjs**
```js
import foo from './foo.mjs';
console.log(foo);
```

**> node bar.mjs**
```text
[class default]
```